docs-next/pages/apis/config.mdx

```diff
@@ -7,20 +7,36 @@ The `keystone-next` CLI expects to find a module named `keystone.ts` with a defa
 ```typescript filename=keystone.ts
 import { config } from '@keystone-next/keystone/schema';
 
-export default config({...});
+export default config({
+  /* ... */
+});
 ```
 
 The `config` function accepts an object representing all the configurable parts of the system:
 
 ```typescript
 export default config({
-  lists: { ... },
-  db: { ... },
-  ui: { ... },
-  server: { ... },
-  session: () => { ... },
-  graphql: { ... },
-  extendGraphqlSchema: { ... },
+  lists: {
+    /* ... */
+  },
```

Comment:
```suggestion
  lists: { /* ... */ },
```
Can we keep these on a single line (is our linter jerking us about here?) II'd really like for this code block to not blow out to three times as many lines as it needs to be 😓